RevenueCatUI/CustomerCenter/Views/PurchaseCardView.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PurchaseInformationCardView.swift
+//
+//  Created by Facundo Menzella on 13/5/25.
+
+import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct PurchaseInformationCardView: View {
+
+    private let title: String
+    private let subtitle: String
+    private let storeTitle: String
+    private let showChevron: Bool
+
+    init(
+        title: String,
+        subtitle: String,
+        storeTitle: String,
+        showChevron: Bool = true
```

Comment:
I'm curious: when will we need to show/hide the chevron?

Reply:
As for now, the detail page will also show this card (without the chevron).

There's a bigger refactor, that consists of removing "Purchase history", and displaying purchase details on the detail screen. Once that's done, we can delete this flag

Reply:
Makes sense! Thanks for the explanation!